src/app/Component/chatroom/chatroom.css

```diff
@@ -110,4 +106,66 @@
     display: flex;
     justify-content: center;
     align-items: center;
-  }
\ No newline at end of file
+  }
+
+  .messagepopup-right{
+    height: 200px;
```

Comment:
use % not px upto 25 px we can use px not more than 25 .If it is more than 25 we need to use %